### PR TITLE
Add section describing horizon setup

### PR DESCRIPTION
**Pull request by lfuelling**
Stuff to do was taken form https://github.com/pixelfed/pixelfed/issues/1283

Followup to: https://comm.network/@lerk/102331984948036051

### CODE REVIEW

File: master/deployment.md [hunk 1]
```diff
@@ -19,4 +19,32 @@ $ php artisan route:cache
 $ php artisan migrate --force
 $ php artisan horizon:purge
 $ php artisan storage:link
+$ php artisan horizon:install
+$ php artisan horizon:assets
 ```
+
+### Background Task Runner
+
+Create the file `/etc/systemd/system/pixelfed-queue.service` with the following content:
```
Comment (trwnh) [inline master/deployment.md]: Not everyone will be running systemd

File: master/deployment.md [hunk 2]
```diff
@@ -19,4 +19,32 @@ $ php artisan route:cache
...
+
+```
+[Unit]
+Description=Pixelfed task queueing via Laravel Horizon
+After=network.target
+Requires=php7.2-fpm.service
```
Comment (trwnh) [inline master/deployment.md]: should be php-fpm, i think? although different distros do different things

File: master/deployment.md [hunk 3]
```diff
@@ -19,4 +19,32 @@ $ php artisan route:cache
...
+Requires=redis.service
+Requires=postgresql.service
```
Comment (trwnh) [inline master/deployment.md]: postgresql is not officially recommended; mysql or mariadb should take its place here

File: master/deployment.md [hunk 4]
```diff
@@ -19,4 +19,32 @@ $ php artisan route:cache
...
+Wants=nginx.service
```
Comment (trwnh) [inline master/deployment.md]: could also be apache

File: master/deployment.md [hunk 5]
```diff
@@ -19,4 +19,32 @@ $ php artisan route:cache
...
+
+[Service]
+Type=simple
+ExecStart=/usr/bin/php /home/pixelfed/artisan horizon
```
Comment (trwnh) [inline master/deployment.md]: indicate artisan needs to be wherever pixelfed was installed

**Comment by trwnh** [comment]
I've left a few comments on particular lines, but I don't know if it should be merged as such -- see also #16 which is a pretty heavy rewrite of the docs (currently WIP).

File: master/deployment.md (hunk 1 above)
Comment (lfuelling) [reply master/deployment.md]: That's right.

File: master/deployment.md (hunk 2 above)
Comment (lfuelling) [reply master/deployment.md]: Not if you're using php7.2 as the docs suggest.

File: master/deployment.md (hunk 3 above)
Comment (lfuelling) [reply master/deployment.md]: That's right, I'll update it.

File: master/deployment.md (hunk 4 above)
Comment (lfuelling) [reply master/deployment.md]: Both is suggested in the docs, I'll add a paragraph about this.

File: master/deployment.md (hunk 5 above)
Comment (lfuelling) [reply master/deployment.md]: There's a paragraph describing this.

**Comment by lfuelling** [comment]
Thanks for your review. 

I'd suggest merging this first and then update #16. Can you give me write access to your fork? I'd do it.

**Pull request closed without merging**